main.cpp

```diff
@@ -52,90 +52,91 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: cat
+//  action 1: the cat scratches
+cat.scratch();
+//  action 2: the cat eats
+cat.eat();
+//  action 3:the cat sleeps
+ cat.sleep();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: footballer
+//  action 1: footballer kicks
+footballer.kick();
+//  action 2: footballer dives
+footballer.dive();
+//  action 3: footballer scores
+ footballer.score();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: pen
+//  action 1: the pen writes
+pen.write();
+//  action 2: the pen leaks
+pen.leak();
+//  action 3: the pen runs out of ink
+pen.runOutOfInk();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:mouse
+//  action 1: mouse clicks
+mouse.click();
+//  action 2: mouse clicks and drags
+mouse.clickAndDrag();
+//  action 3: mouse moves
+mouse.move();
+//  5) alarmClock
+//  Noun: alarm clock
+//  action 1:alarm clock ticks
+alarmClock.tick();
+//  action 2:alarm clock turns hour hand
+alarmClock.turnHourHand();
+//  action 3:alarm clock rings
+alarmClock.ring();
+//  6) 
+//  Noun: star
+//  action 1: star shines
+star.shine();
+//  action 2:star twinkles
+star.twinkle();
+//  action 3:star implodes
+star.implode();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: octopus
+//  action 1:octopus swims
+octopus.swim();
+//  action 2:octopus changes skin colour
+octopus.changeSkinColour();
+//  action 3:octopus retracts tentacle
+octopus.retractTentacle();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  10)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: sun
+//  action 1:sun shines
+sun.shine();
+//  action 2:sun rotates
+sun.rotate();
+//  action 3:sun emits flare
+sun.emitFlare();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**REPLACE** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.
**Do not** create an 11th noun.